src/mrnet/core/reactions.py

```diff
@@ -1860,6 +1975,134 @@ def from_dict(cls, d):
         return reaction
 
 
+def general_graph_rep(reaction: Reaction) -> nx.DiGraph:
+    """
+    A method to convert a reaction type object into a general graph representation.
+    Can handle reactions with arbitrary numbers of reactants and products
+    Args:
+       :param reaction:(any of the reaction class object, ex. RedoxReaction,
+       IntramolSingleBondChangeReaction)
+    """
+    # Create the graph object, and define/call appropriate data
+    graph = nx.DiGraph()
+    rxn_type_A = reaction.rxn_type_A
+    rxn_type_B = reaction.rxn_type_B
+    energy_A = reaction.energy_A
+    energy_B = reaction.energy_B
+    reaction.free_energy()
```

Comment:
Why would you always re-define it based on 298? This, again, feels like it should be a user-defined parameter. And further, maybe you should only re-define the free energy if it's None, or if the user requests it?

Reply:
So the free_energy cannot always be calculated upon initialization-for example, if the electron_free_energy is not defined at the time of initialization, this can throw a wrench in the works. For now, the reason that the method defaults to 298 is because the graph_representation method doesn't take in a temperature. This is a relatively simple edit, and if you would like to me to do so, I can do that in a future PR.